.github/workflows/run-k8s-tests-on-garm.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
         run: bash tests/integration/kubernetes/gha-run.sh run-tests
   
       - name: Collect artifacts ${{ matrix.vmm }}
-        run: bash tests/integration/kubernetes/gha-run.sh collect-artifacts
+        run: bash tests/integration/kubernetes/gha-run.sh collect-artifacts || true
```

Comment:
Hi @GabyCT ! Thanks for promptly working on this one!

One question that I have and I forgot to mention on the issue is whether the next step `Archive artifacts ${{ matrix.vmm }}` is going to fail in case /tmp/artifacts doesn't exist.

ah, also, while here, I think we want to have this step ` if: always()` so it runs in case the test fails.

Reply:
@wainersm so I added the if always step